src/main/java/com/ericsson/ei/handlers/RmqHandler.java

```diff
@@ -176,17 +192,6 @@ public SimpleMessageListenerContainer bindToQueueForRecentEvents(ConnectionFacto
         return container;
     }
 
-    public String getQueueName() {
-        String durableName = queueDurable ? "durable" : "transient";
-        return domainId + "." + componentName + "." + consumerName + "." + durableName;
-    }
-
-    public String getWaitlistQueueName() {
-
-        String durableName = queueDurable ? "durable" : "transient";
-        return domainId + "." + componentName + "." + consumerName + "." + durableName + "." + waitlistSufix;
-    }
-
     @Bean
     public RabbitTemplate rabbitMqTemplate() {
```

Comment:
Should we have some javadoc on either this method or the one called publishObjectTowaitlistQueue? Just to explain that the default set routing key is only used when EI itself sends events again onto the message bus ...